backend/src/nodes/properties/inputs/pytorch_inputs.py

```diff
@@ -57,12 +50,16 @@ def __init__(
             ),
         )
         if torch is not None:
-            self.associated_type = Union[SRModelDescriptor, RestorationModelDescriptor]
+            self.associated_type = ImageModelDescriptor
 
-    def enforce(self, value: object):
+    def enforce(self, value: ModelDescriptor):
```

Comment:
This is not how `enforce` works. The whole point of `enforce` is to verify that a value is of a certain type, so you can't just declare that it already is that type.

Reply:
Whoops. I'll fix that.